test/test-remote/testutils/logs.ts

```diff
@@ -66,10 +66,16 @@ export async function queryLoki(baseUrl: string, queryParams: URLSearchParams) {
   const headers = enrichHeadersWithAuthToken(url, {});
 
   const options = {
-    retry: 0,
+    // Allow up to two retries in the event of spurious timeouts or similar errors.
+    retry: 2,
```

Comment:
It's fine, but I am not sure about the logging quality around the internal retries right now. Maybe we should let said loop body take care of the retries (the rational behind putting `retry: 0`) so that the build log contains all detail of what happened. Of course, with `got`-internal retrying we can still achieve great logging. I tried to work towards that here: https://github.com/opstrace/opstrace/blob/90906e24da0b5a677186d6e53ba0070184eafdf4/lib/utils/src/httpclient.ts#L43 -- but in the context of test-remote, we don't have that kind of control yet. I am saying: we might not see in the build log when this internal retrying had to hit in.